### ISSUE TITLE
Questions about the gnn_models_sag.py

### ISSUE THREAD

**Issue by wan-nie** [zqgao22/HIGH-PPI#3]
Hello Ziqi, I checked the `gnn_models_sag.py` file and observed that the model architectures are quite different from the models depicted in **Supplementary Fig. 1** of the paper.

Regarding BGNN, several discrepancies are present, including the feature dimension, activation function type, GCN block count, the order of activation function and batch normalization layers, and additional `GCNConv` modules following SAG pooling, which contradicts Supplementary Fig. 1. Similarly, in TGNN, replacing the `feature_fusion` argument with 'concat' (i.e., concatenation used in the paper) is insufficient, and I must modify the feature dimension of `fc2`. I may have overlooked other discrepancies.

Based on the above findings, it appears that there may be a final version of the model structure. Would it be possible for you to release the final version of the `gnn_models_sag.py` file so that we can directly implement the best model architecture as described in the paper? Thank you!

**Comment by AcetylCoALab** [comment]
I agree with Dr. wan-nie's point. I look forward to the authors releasing the final version of the paper, which will be very helpful for us to reproduce the results

**Comment by zqgao22** [comment]
Sorry for misleading you with our existing model architecture. We later found that the existing version of the model can achieve greater robust performance, so it temporarily replaced the version presented in the paper. We will immediately update a final version that matches the information shown in our paper for you to reproduce the results.
Thereafter, for new model architectures that may improve performance, we will simultaneously update and retain the original model.

**Comment by wan-nie** [comment]
Glad to hear about that, it will be helpful! ; )

**Comment by suice07** [comment]
> Sorry for misleading you with our existing model architecture. We later found that the existing version of the model can achieve greater robust performance, so it temporarily replaced the version presented in the paper. We will immediately update a final version that matches the information shown in our paper for you to reproduce the results. Thereafter, for new model architectures that may improve performance, we will simultaneously update and retain the original model.

notice that in the model_test.py, the network GIN_Net2 was imported from the gnn_models_sag.py, is the GIN_Net2 the same as the GIN?
